openprescribing/matrixstore/open.py

```diff
@@ -0,0 +1,53 @@
+import os.path
+import sqlite3
+
+from .serializer import deserialize
+
+
+class MatrixStore(object):
```

Comment:
Is `open` the right name for this module?

Reply:
No, I don't think it is. But then at the time I couldn't think of the right name and I didn't want to get stuck so I just went with the first thing I thought of. Better suggestions are very welcome. `matrixstore is the natural one, but then `from matrixstore.matixstore import MatrixStore` isn't great. Maybe `connection` or `database_wrapper`?

Reply:
How about `wrapper` or similar, but also import it in `__init__` so you can `from matrixstore import MatrixStore`?

I'm not normally a fan of fiddling namespaces in `__init__` but as the primary entry point for matrixstore operations it may make sense

Reply:
+1 for `connection`.

And as the `MatrixStore` class is the primary way we'll interact with the database, `from matrixstore import MatrixStore` feels sensible.

Reply:
I don't actually anticipate we'll be importing this directly very much as the primary interface point for the rest of the code will be the `matrixstore.db` module which comes in the next PR. (In the same way that we don't import Django's connection wrappers but just use the managed connections.) I wanted to avoid `matrixstore.matrixstore` for the sake of confusion and inelegance more than the sake of length. So I'd rather avoid messing about with `__init__.py` unless we really need to.

I'm happy with either `wrapper` or `connection`. Seb, as technical director and keeper of the bike shed paint brushes, do you want to make the casting vote?

Reply:
`connection`